charts/qdrant/templates/tests/test-db-interaction.yaml

```diff
@@ -53,11 +59,11 @@ data:
     QDRANT_COLLECTION="test_collection"
     {{- range .Values.service.ports }}
     {{- if eq .name "http" }}
-    echo "Connecting to {{ include "qdrant.fullname" $root }}.{{ $namespace }}:{{ .port }}"
-    QDRANT_URL="http://{{ include "qdrant.fullname" $root }}.{{ $namespace }}:{{ .port }}"
+    echo "Connecting to {{ include "qdrant.fullname" $root }}:{{ .port }}"
+    QDRANT_URL="http://{{ include "qdrant.fullname" $root }}:{{ .port }}"
```

Comment:
This doesn't really have an effect, but using just the pod name as the DNS name saves us an extra DNS query, since `.<namespace>.svc.cluster.local` is always appended to the query first, then `.svc.cluster.local`, then `.cluster.local`, etc. until one of them works.